lib/health_monitor/providers/database.rb

```diff
@@ -11,7 +11,7 @@ def check!
         failed_databases = []
 
         ActiveRecord::Base.connection_handler.connection_pool_list(:all).each do |cp|
-          cp.connection.execute('SELECT 1')
+          cp.lease_connection.execute('SELECT 1')
```

Comment:
This should be conditioned by the version of Rails, since it will likely fail for older versions. You should add Rails 8 to the `Appraisal` test matrix so that we can be sure that everything works.

Reply:
@lbeder this hasn't been addressed? Seeing this error now in a Rails 7 app

Reply:
Added PR https://github.com/lbeder/health-monitor-rails/pull/132